sites/cheerpx/src/content/docs/20-faq.md

```diff
@@ -80,3 +80,20 @@ Our cloud disk backend is based on WebSocket and distributed across a global CDN
 While v86 might use an open proxy to the internet, this approach is not feasible for CheerpX due to its scale of use. An open proxy can pose significant security and abuse risks when used at scale.
 
 Instead, CheerpX supports Tailscale-based networking, which can accommodate many use cases. It's important to note that developers using CheerpX are responsible for implementing appropriate security measures and preventing potential abuse.
+
+## Missing cross-origin isolation
```

Comment:
This is not part of the input/output guide, please try to keep PRs on-topic, so that work can be merged independently.

Reply:
I've removed the info, going to place it in another branch